scrape/target.go

```diff
@@ -492,3 +502,36 @@ func TargetsFromGroup(tg *targetgroup.Group, cfg *config.ScrapeConfig) ([]*Targe
 	}
 	return targets, failures
 }
+
+// buildTargetLabels builds final target labels based on the given target label set, target group label set and scrape config.
+func buildTargetLabels(targetLSet model.LabelSet, targetGroupLSet model.LabelSet, cfg *config.ScrapeConfig) labels.Labels {
```

Comment:
```suggestion
func buildTargetLabels(targetLSet, targetGroupLSet model.LabelSet, cfg *config.ScrapeConfig) labels.Labels {
```

Reply:
ah yeah, cool it fixed the linter. Thanks @roidelapluie